ShiftLoggerApi/ShiftLoggerApi.sln

```diff
@@ -0,0 +1,16 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ShiftLoggerApi", "ShiftLoggerApi\ShiftLoggerApi.csproj", "{6EB8E8DD-39A8-4477-98E1-301279073E3C}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{6EB8E8DD-39A8-4477-98E1-301279073E3C}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{6EB8E8DD-39A8-4477-98E1-301279073E3C}.Debug|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
You'll need to create these two projects in side a KimFom.ShiftLogger project. So the hierarchy will be: ParentFolder > KimFom.ShiftLogger > Api, UI

Reply:
done